selection_tools/include/Particle.hpp

```diff
@@ -29,8 +29,12 @@ class Particle
   double eta();
   double momentum() const;
   double pt() const;
-  
-  
+  double Px() const;
+  double M() const;
+  double E() const;
```

Comment:
Could you please rename these to "mass" and "energy" respectively (to avoid any confusion in the future).

Reply:
This is a bit temporary, I can't name them mass or energy since we already have them for a particle called as particle->mass and particle->energy, but I realized that these values are different from four_vector.M(), so I was testing this out and named the functions the same as for TLorentzVector